plugins/com.google.cloud.tools.eclipse.appengine.deploy.test/src/com/google/cloud/tools/eclipse/appengine/deploy/standard/StandardDeployPreferencesTest.java

```diff
@@ -0,0 +1,45 @@
+package com.google.cloud.tools.eclipse.appengine.deploy.standard;
+
+import static org.junit.Assert.*;
```

Comment:
google style is not to use wildcard imports: https://google.github.io/styleguide/javaguide.html#s3.3-import-statements